install_config/install/advanced_install.adoc

```diff
@@ -1694,6 +1842,70 @@ If for any reason the installation fails, before re-running the installer, see
 xref:installer-known-issues[Known Issues] to check for any specific
 instructions or workarounds.
 
+[[running-the-advanced-installation-cns-gluster-persistent-storage]]
+=== Completing Containerized GlusterFS Persistent Storage Configuration
+
+If you followed the steps in
+xref:advanced-install-containerized-glusterfs-persistent-storage[Configuring Containerized GlusterFS Persistent Storage], run the following steps manually
+after the cluster installation has completed succesfully to complete your
+GlusterFS configuration:
+
+. Update the *_/etc/dnsmasq.conf_* file on all masters to enable proper DNS
+communication between the masters, nodes, and the Heketi server pods routes. Add
+the following line replacing with appropriate values:
++  
+----
+address=/.<openshift_master_default_subdomain>/<router_IP>
+----
++
+For example:
++
+----
+address=/.cloudapps.example.com/172.18.23.24
+----
+
+. Restart the `dnsmasq` service:
++
+----
+# systemctl restart dnsmasq
+----
+
+. From a master, verify that the route is working.
+
+.. Verify that the route was created:
++
+----
+# oc get routes
+NAME            HOST/PORT                                     PATH           SERVICES   PORT   TERMINATION   WILDCARD
+heketi-storage  heketi-storage-default.cloudapps.example.com  heketi-storage <all>             None
+----
+
+.. Curl the route:
++
+----
+# curl http://heketi-storage-default.cloudapps.example.com/hello
+Hello from Heketi.
+----
+
+. Verfiy that the GlusterFS `StorageClass` was created:
++
+----
+# oc get storageclass
+NAME                  TYPE
+glusterfs-storage     kubernetes.io/glusterfs
+----
++
+[NOTE]
+====
+The default name for the route will be `heketi-storage` unless this was
+overridden using the `openshift_glusterfs_storage_name` variable in the
+inventory file.
```

Comment:
It's actually `heketi-glusterfs-route`, where the `glusterfs` part can be overridden with `openshift_storage_glusterfs_name` and `openshift_storage_glusterfs_registry_name`.